<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>621</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>iMes</string>
      </property>
      <widget class="QScrollArea" name="scrollArea_2">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>521</width>
         <height>251</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>519</width>
          <height>249</height>
         </rect>
        </property>
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>-9</y>
           <width>521</width>
           <height>261</height>
          </rect>
         </property>
        </widget>
       </widget>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>30</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>60</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>90</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>120</y>
         <width>151</width>
         <height>21</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>当前工单</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>60</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>资料名</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>90</y>
         <width>21</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>IP</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>通讯端口</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>连接iMes</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_5">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>断开</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_6">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>参数设置</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_Modify_Password">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>240</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>修改密码</string>
       </property>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="font">
       <font>
        <family>Agency FB</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="title">
       <string>AOI</string>
      </property>
      <widget class="QTextBrowser" name="textBrowser">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>20</y>
         <width>521</width>
         <height>251</height>
        </rect>
       </property>
      </widget>
      <widget class="QRadioButton" name="radioButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>25</y>
         <width>101</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>ActProgTpye</string>
       </property>
      </widget>
      <widget class="QScrollArea" name="scrollArea">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>19</y>
         <width>521</width>
         <height>251</height>
        </rect>
       </property>
       <property name="widgetResizable">
        <bool>true</bool>
       </property>
       <widget class="QWidget" name="scrollAreaWidgetContents">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>519</width>
          <height>249</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QRadioButton" name="radioButton_2">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>25</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>ActUtlTpye</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>80</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>连接收板机</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>120</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>160</y>
         <width>161</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>地址段</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>160</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>返回值</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>通信模式</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>开始</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>200</y>
         <width>61</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Agency FB</family>
         <pointsize>14</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>停止</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>240</y>
         <width>81</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string>倒计时:</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>180</x>
         <y>240</y>
         <width>72</width>
         <height>31</height>
        </rect>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
      <widget class="QCheckBox" name="checkBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>57</y>
         <width>131</width>
         <height>19</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>快速机模式</string>
       </property>
      </widget>
      <zorder>scrollArea</zorder>
      <zorder>textBrowser</zorder>
      <zorder>radioButton</zorder>
      <zorder>radioButton_2</zorder>
      <zorder>pushButton</zorder>
      <zorder>lineEdit</zorder>
      <zorder>lineEdit_2</zorder>
      <zorder>label</zorder>
      <zorder>label_2</zorder>
      <zorder>label_3</zorder>
      <zorder>pushButton_2</zorder>
      <zorder>pushButton_3</zorder>
      <zorder>label_8</zorder>
      <zorder>label_9</zorder>
      <zorder>checkBox</zorder>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
